packages/firestore/src/remote/serializer.ts

```diff
@@ -167,12 +167,13 @@ export class JsonProtoSerializer {
    * our generated proto interfaces say Int32Value must be. But GRPC actually
    * expects a { value: <number> } struct.
    */
-  private toInt32Value(val: number | null): number | undefined {
-    if (!typeUtils.isNullOrUndefined(val)) {
+  private toInt32Value(val: number | null): number | null {
```

Comment:
You changed the return type (to `null`) but need to update the comment to match.

Reply:
oops, good catch!